DataDictionary.md

```diff
@@ -33,6 +33,7 @@
 3. Criminal Actions - Severe action reserved for facilities intentionally committed
 
 ## Penalty
+* Penalties most often incur a fee as a result of noncompliance with enviromental regulations
```

Comment:
"Penalties most often incur a fee" makes it sound like the penalties are fined. maybe "result in" rather than "incur"?